cogs/welcome/welcome.py

```diff
@@ -5,10 +5,14 @@
 import asyncio
 import discord
 import logging
+import random
 
-from redbot.core import Config, checks, commands
+from redbot.core import Config, checks, commands, data_manager
```

Comment:
don't see this import being used, can we revert this?

Reply:
done